modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/ags_linear_solver.h

```diff
@@ -12,55 +12,38 @@ namespace lbs
 {
 
 /**Linear Solver specialization for Within GroupSet (WGS) solves.*/
-class AGSLinearSolver : public LinearSolver
+class AGSLinearSolver
 {
+private:
+  std::shared_ptr<LinearSolverContext> context_ptr_;
+  int maximum_iterations_;
+  double tolerance_;
+  bool verbose_;
```

Comment:
I forgot whether we decided on any format for class layout, but attributes are at the bottom of class definitions in most cases.

Reply:
Fixed.